subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/reservation/adb/Device.kt

```diff
@@ -35,15 +42,15 @@ class Device(
     )
 
     fun disconnect(): Try<String> = runCommand(
-        command = "adb disconnect $serial",
+        command = "$adb disconnect $serial",
```

Comment:
What's the point? How did it work before?

Reply:
Before you could use adb if you had path to adb in $PATH. Now you need to declare $ANDROID_HOME explicitly